### ISSUE TITLE
lmranks returns y that differs from frank

### ISSUE THREAD

**Issue by danielwilhelm** [danielwilhelm/R-CS-ranks#26]
When using lmranks with the options y=TRUE and omega=omega, then the y attribute returns ranks that differ from frank(..., increasing=TRUE, omega=omega). Why is that? Is this expected?

**Comment by MrDomani** [comment]
Yes, this was done on purpose.
The reason is frank function does not return values coherent with R_x definition in the paper. In paper, R_x is $\omega \sum 1(x < x_i) + (1-\omega) \sum 1(x <= x_i)$. In the code of irank (originally created by you), 1-omega is added to the result. I assume, that the intention was to fix top observations frank as 1 (or bottom observations as 0).
Currently, r() in lmranks with default arguments returns the same values as R's function ecdf. 
Sorry for not mentioning this earlier.

**Comment by danielwilhelm** [comment]
The rank definition we should use is the one from the paper, equation (7), i.e. $\hat{R}_X(x)$, not $R_X(x)$. Asymptotically, they are the same, but for fixed $n$, they may differ when there are ties.

For $\omega=1$, the definition in (7) is equal to the empirical pdf (and thus equal to the R command ecdf()). This should be the default in frank() and in lmranks(), using increasing=TRUE and omega=1.

**Comment by danielwilhelm** [comment]
Ok, now I see how lmranks() computes the ranks. For omega=1, it produces the correct ranks, which are equal to frank(..., omega=1, increasing=TRUE) and to equation (7). This is because in this default case, the extra $\frac{1-\omega}{n}$ is zero. When the omega argument in lmranks() takes a value different from 1, then the rank used by lmranks() does not equal the definition in frank() nor the definition in (7) of the paper.

In conclusion, I'd like lmranks() to use frank(..., omega=omega, increasing=TRUE). That's the correct rank to use.
